### PR TITLE
Copter: Payload Place: Change PLDP_RNG_MIN to PLDP_RNG_MAX

### PR DESCRIPTION
**Pull request by lthall**
I noticed that I messed up the name of PLDP_RNG_MIN. This PR fixes this problem by changing PLDP_RNG_MIN to PLDP_RNG_MAX

### CODE REVIEW

**Comment by rmackay9** [comment]
Hi, the parameter descriptions a few lines above will also need to be updated I think.

**Comment by rmackay9** [comment]
We will need to update this wiki page as well (Henry or I will take care of this I suspect). https://ardupilot.org/copter/docs/mission-command-list.html#payload-place

**Review by rmackay9 — APPROVED** [review]
LGTM, thanks!

File: ArduCopter/Parameters.cpp [hunk 1]
```diff
@@ -1176,13 +1176,13 @@ const AP_Param::GroupInfo ParametersG2::var_info2[] = {
     // @User: Standard
     AP_GROUPINFO("PLDP_THRESH", 1, ParametersG2, pldp_thrust_placed_fraction, 0.9),
 
-    // @Param: PLDP_RNG_MIN
+    // @Param: PLDP_RNG_MAX
     // @DisplayName: Payload Place minimum range finder altitude
```
Comment (tridge) [inline ArduCopter/Parameters.cpp:1180]: should be maximum in displayname and description

File: ArduCopter/mode_auto.cpp [hunk 2]
```diff
@@ -1301,13 +1301,13 @@ void PayloadPlace::run()
...
-        } else if (is_positive(g2.pldp_range_finder_minimum_m)) {
+        } else if (is_positive(g2.pldp_range_finder_maximum_m)) {
             if (!copter.rangefinder_state.enabled) {
                 // abort payload place because rangefinder is not enabled
                 state = State::Ascent_Start;
                 gcs().send_text(MAV_SEVERITY_WARNING, "%s PLDP_RNG_MIN set and rangefinder not enabled", prefix_str);
                 break;
-            } else if (copter.rangefinder_alt_ok() && (copter.rangefinder_state.glitch_count == 0) && (copter.rangefinder_state.alt_cm > g2.pldp_range_finder_minimum_m * 100.0)) {
+            } else if (copter.rangefinder_alt_ok() && (copter.rangefinder_state.glitch_count == 0) && (copter.rangefinder_state.alt_cm > g2.pldp_range_finder_maximum_m * 100.0)) {
                 // range finder altitude is above minimum
```
Comment (tridge) [inline ArduCopter/mode_auto.cpp:1311]: maximum

**Review by tridge — CHANGES_REQUESTED** [review]

File: ArduCopter/Parameters.cpp (hunk 1 above)
Comment (rmackay9) [reply ArduCopter/Parameters.cpp]: fixed

File: ArduCopter/mode_auto.cpp (hunk 2 above)
Comment (rmackay9) [reply ArduCopter/mode_auto.cpp]: fixed

File: ArduCopter/mode_auto.cpp [hunk 3]
```diff
@@ -1301,14 +1301,14 @@ void PayloadPlace::run()
             // thrust is above minimum threshold
             place_start_time_ms = now_ms;
             break;
-        } else if (is_positive(g2.pldp_range_finder_minimum_m)) {
+        } else if (is_positive(g2.pldp_range_finder_maximum_m)) {
             if (!copter.rangefinder_state.enabled) {
                 // abort payload place because rangefinder is not enabled
                 state = State::Ascent_Start;
                 gcs().send_text(MAV_SEVERITY_WARNING, "%s PLDP_RNG_MIN set and rangefinder not enabled", prefix_str);
```
Comment (peterbarker) [inline ArduCopter/mode_auto.cpp:1308]: This line also needs fixing

**Review by peterbarker — CHANGES_REQUESTED** [review]

**Review by peterbarker — APPROVED** [review]

**Pull request merged**